lib/logic/child/task_instance/task_instance_cubit.dart

```diff
@@ -28,8 +28,9 @@ class TaskInstanceCubit extends Cubit<TaskInstanceState> {
 	final NotificationService _notificationService = GetIt.I<NotificationService>();
 	final UIDataAggregator _dataAggregator = GetIt.I<UIDataAggregator>();
 
-	TaskInstanceCubit(this._taskInstanceId, this._activeUser) : super(TaskInstanceStateInitial());
+	TaskInstanceCubit(this._taskInstanceId, this._activeUser, this.uiPlanInstance) : super(TaskInstanceStateInitial());
 
+	UIPlanInstance uiPlanInstance;
```

Comment:
This is much better that keeping it in the UI layer but we can do even better. Cubit state is the place to keep all the data, UI should only ever call functions on cubits, not access any of the member fields. You can add a `UIPlanInstance` to `TaskInstanceStateInitial` and access it from there.